### DISCUSSION TITLE
Looks like Thermia portal is down

### DISCUSSION THREAD

**Discussion by fclauson** [klejejs/ha-thermia-heat-pump-integration#88] [General]
You can login in but there is no data from some time over night

**Comment by klejejs** [comment]
@fclauson I converted this to a discussion, so other people can learn about it and participate in it as well but closed the issue itself, because, as I mentioned in the comment, this is a 3rd party integration and there is nothing I can do about Thermia problems.

**Comment by fclauson** [comment]
thanks - sorry for doing it in the wrong place
I have raised a request to info@thermia.se asking them if they are aware and when can we expect it to be back up again
